config/initializers/assets.rb

```diff
@@ -5,8 +5,6 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
TODO(I will do it in this PR)
I couldn't decide whether we accept this change or not.
Because we seems to use both `sprockets` and `webcapkcer` now.
I want to make sure `sprockets` don't handle files under `app/javascript`.

Reply:
We can accept this change of the following reason.
`Webpacker` handles JavaScript files under `app/javascript` and show them on view files using `javascript_pack_tag`.
It means we can delete this code because we don't have to handle them using `Sprockets`

https://github.com/ranguba/ranguba/blob/0a7fb8e9d6e2a26158e30c4b776890b92bc6ce14/app/views/layouts/application.html.erb#L13-L14